bin/p2-bootstrap/bootstrap.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 	"time"
 
-	"github.com/square/p2/pkg/intent"
+	"github.com/square/p2/pkg/p2consul"
```

Comment:
I like `store`. After resolving any other comments, I'll do a big rebase to rename the package and all the references to its old name.